### DISCUSSION TITLE
Generating a thesis like pdf with juypter book

### DISCUSSION THREAD

**Discussion by Leon0402** [executablebooks/meta#520] [Q&A]
Hi, 

I want to generate some sort of bachelor thesis with jupyter book. The requirements aren't to strong, but basic things like a toc with links, citations, bibliography, page numbers etc. should work. 

I had a look at converting with the builder pdflatex. It generally works and provides things like toc, page numbering and stuff like this. But it has some problems: For instance no svg rendering and not much options to customize the theme and such stuff. 

The pdfhtml looks much more customizable as everything can by styled with css, which sounds great and supports things like svg images. But the "default look" looks - well - like the html as a pdf. No page numberings, no tocs, no bibliography. No "thesis" like theme. 
Are these sort of things even possible with pdfhtml due to using html before converting to pdf? 

Would it be best to try to make html work or pdf? Any solutions to any of the problems?

**Comment by TomLav** [comment]
Hei Leon,

I guess you moved on this since your post in Oct 2021, but I am interested in a related issue (preparing "pretty" PDFs from a JupyterBook).

How did you approach this in the end? Do you have tips and recipes to share?

Thanks,
Thomas
